windows/README.md

```diff
@@ -26,7 +26,7 @@ tidal login
     - Specify `-UserName` parameter value
     - Store the password securely by running `.\save_password.ps1` script
 
-4) Ensure you have a file that has a list of all the hostnames you want to scan. By default, Machine Stats looks for `servers.txt`, but you can also specify any custom location with `-ServersPath` parameter. The hosts will need to be accessible via your network connection from the machine that you run this from.
+4) Ensure you have a file that has a list of all the private IP addresses you want to scan. For analyzing machines over the internet, you can use the public IPv4 DNS name. By default, Machine Stats looks for `servers.txt`, but you can also specify any custom location with `-ServersPath` parameter. The hosts will need to be accessible via your network connection from the machine that you run this from.
```

Comment:
"Ensure you have a file"... is it any type of file? or it must be a myfile.txt ?

I find it confusing especially when I read the next paragraph.  To me, it sounds like I can use the file generated by the platform. That file is named something like this `servers_2022-03-18.xlsx`.
Would that work?

Reply:
Good question Mario, I'm not sure. I'll give it a go with some different file types when I get the chance.